tests/test_message.py

```diff
@@ -482,6 +482,46 @@ async def make_assertion(*args, **kwargs):
         if reply_parameters is None or reply_parameters.message_id != 42:
             pytest.fail(f"reply_parameters is {reply_parameters} but should be 42")
 
+    @staticmethod
+    async def check_thread_id_parsing(
+        message: Message, method, bot_method_name: str, args, monkeypatch
+    ):
+        """Used in testing reply_* below. Makes sure that meassage_thread_id is parsed
+        correctly."""
+
+        async def make_assertion(*args, **kwargs):
+            return kwargs.get("message_thread_id")
+
+        monkeypatch.setattr(message.get_bot(), bot_method_name, make_assertion)
+
+        message.message_thread_id = None
+        message_thread_id = await method(*args)
+        assert message_thread_id is None
+
+        message.message_thread_id = 99
+        message_thread_id = await method(*args)
+        assert message_thread_id == 99
+
+        message_thread_id = await method(*args, message_thread_id=50)
+        assert message_thread_id == 50
+
+        if bot_method_name != "send_chat_action":
```

Comment:
```suggestion
        if bot_method_name == "send_chat_action":
            return
```

Avoids unnecessary nesting :)